### DISCUSSION TITLE
Bot verwendung

### DISCUSSION THREAD

**Discussion by BlueDragon4251** [ScootKit/CustomDCBot#143] [Questions] [answered]
Kann ich den bot herunterladen und verwenden wenn ja wo sind die Datei die ich konfigurieren muss sowie die main Datei? Oder die einzelnen module

**Comment by SCDerox** [comment, marked as answer]
Sorry, but the official language of this repository is English. Please keep your discussions in English. Please follow the [Installation-Instructions](https://github.com/ScootKit/CustomDCBot#installation) carefully. You need to run `npm run generate-config` to install dependencies required for the configuration generation. If you don't want to worry about setup and configuration, we recommend using [SCNX](https://scnx.xyz/bot).

**Reply by BlueDragon4251** [reply]
Sorry I only found out now but thanks for the quick response

**Reply by BlueDragon4251** [reply]
![image](https://github.com/ScootKit/CustomDCBot/assets/136262109/8bf9eb2f-5a71-4c99-8a3c-86ad670fd5ef)
what should i do here? this comes up everytime

**Reply by SCDerox** [reply]
Disable the SCNX Integration flag? You can't use the SCNX Integration flag if you are not running on SCNX infrastructure.

**Reply by BlueDragon4251** [reply]
still:
![image](https://github.com/ScootKit/CustomDCBot/assets/136262109/936a8d83-4d64-4c40-94d2-2c7ad617e14f)

**Reply by SCDerox** [reply]
You can't use the SCNX integration flag if you are not running on SCNX infrastructure. Please ensure you are running in a supported environment and do not add environment flags, such as the scnx enabled flag when running the CustomBot.

**Reply by BlueDragon4251** [reply]
but the flag is disabled anyway anyway the error message comes

**Reply by SCDerox** [reply]
Thanks for bringing this to my attention. This will be fixed in a future updated, but isn't a priority. For now, try disabling the module that causes this issue.

**Reply by BlueDragon4251** [reply]
but i dont know what module causes it

**Reply by SCDerox** [reply]
Please do not remove the answered mark, as your original issue has been answered. You could try disabling all modules and enabling them one-by-one to investigate this, alternatively, look for the elementLimits flag on selected configuration files to determine module causation.

**Reply by BlueDragon4251** [reply]
I've found it it is the twitch-notification module pls fix that in the coming update thx

**Reply by jateute** [reply]
> I've found it it is the twitch-notification module pls fix that in the coming update thx

I‘ll try to fix it soon. Because it’s just a matter of changing the config file, I might be able to make a PR today when I‘m back at my PC.

**Reply by SCDerox** [reply]
This is actually part of core, the config file option should be ignored on non-scnx builds
